apps/condo/domains/ticket/hooks/useImporterFunctions.ts

```diff
@@ -0,0 +1,224 @@
+        { name: IsResidentTicketLabel, type: 'string', required: false, label: IsResidentTicketLabel },
+        { name: PhoneLabel, type: 'string', required: false, label: PhoneLabel },
+        { name: FullNameLabel, type: 'string', required: false, label: FullNameLabel },
+        { name: DetailsLabel, type: 'string', required: true, label: DetailsLabel },
+        { name: OldTicketNumberLabel, type: 'string', required: false, label: OldTicketNumberLabel },
+        { name: CreatedAtLabel, type: 'custom', required: false, label: CreatedAtLabel },
+    ]
+
+    const ticketNormalizer: RowNormalizer = async (row) => {
+        const [address, unitName, isResidentTicket, phone, fullName, details, oldTicketNumber, createdAt] = row
+        const addons = {
+            address: null,
+            propertyId: null,
+            phone: null,
+            isValidPhone: null,
+            fullName: null,
+            details: null,
+            isResidentTicket: null,
+            contacts: null,
+            unitName: null,
+            unitType: null,
+            createdAt: null,
+            isEmptyDetails: null,
+            oldTicketNumber: null,
+        }
+
+        const suggestionOptions = await addressApi.getSuggestions(String(address.value))
+        const suggestion = get(suggestionOptions, ['suggestions', 0])
+        if (suggestion) {
+            addons.address = suggestion.value
+
+            const properties = await searchProperty(client, {
+                address: suggestion.value,
+                organization: { id: userOrganizationId },
+            }, undefined)
+
+            addons.propertyId = !isEmpty(properties) ? properties[0].value : null
+        }
+
+        const phoneNumber = String(get(phone, 'value', '')).trim()
+        addons.phone = phoneNumber || null
+        addons.isValidPhone = Boolean(normalizePhone(phoneNumber, true))
+        addons.createdAt = createdAt.value ? String(createdAt.value) : ''
+        addons.fullName = String(get(fullName, 'value', '')).trim()
+        addons.details = String(get(details, 'value', '')).trim()
+        addons.oldTicketNumber = String(get(oldTicketNumber, 'value', '')).trim()
+        addons.isResidentTicket = normalizeIsResidentTicket(String(get(isResidentTicket, 'value', '')).trim())
+        addons.unitName = unitName.value ? String(unitName.value).trim() : null
+        addons.unitType = addons.unitName ? 'flat' : null
+        addons.isEmptyDetails = Boolean(String(get(details, 'value', '')).trim())
+
+        const { data: { objs } } = await searchContacts(client, {
+            organizationId: userOrganizationId,
+            propertyId: addons.propertyId,
+            unitName: addons.unitName,
+            unitType: addons.unitType,
+        })
+        addons.contacts = objs
+
+        return Promise.resolve({ row, addons })
+    }
+
+    const ticketValidator: RowValidator = (row) => {
+        if (!row) return Promise.resolve(false)
```

Comment:
You can change declaration to async - then you can use simple return instead of Promise.resolve